examples/subscriptions/app/routes/products.$handle.tsx

```diff
@@ -240,124 +418,175 @@ function SellingPlanPrice({
   );
 }
 
-/**
-  Render the price of a product that does not have selling plans
-**/
-function ProductVariantPrice({
-  selectedVariant,
+// Update as you see fit to match your design and requirements
+function SellingPlanGroup({
+  sellingPlanGroup,
 }: {
-  selectedVariant: ProductVariantFragment;
+  sellingPlanGroup: SellingPlanGroup;
 }) {
-  return selectedVariant?.compareAtPrice ? (
-    <>
-      <p>Sale</p>
-      <br />
-      <div className="product-price-on-sale">
-        {selectedVariant ? <Money data={selectedVariant.price} /> : null}
-        <s>
-          <Money data={selectedVariant.compareAtPrice} />
-        </s>
-      </div>
-    </>
-  ) : (
-    selectedVariant?.price && <Money data={selectedVariant?.price} />
+  return (
+    <div key={sellingPlanGroup.name}>
+      <p className="mb-2">
+        <strong>{sellingPlanGroup.name}:</strong>
+      </p>
+      {sellingPlanGroup.sellingPlans.nodes.map((sellingPlan) => {
+        return (
+          <Link
+            key={sellingPlan.id}
+            prefetch="intent"
+            to={sellingPlan.url}
+            className={`border inline-block p-4 mr-2 leading-none py-1 border-b-[1.5px] hover:no-underline cursor-pointer transition-all duration-200
+                  ${
+                    sellingPlan.isSelected
+                      ? 'border-gray-500'
+                      : 'border-neutral-50'
+                  }`}
```

Comment:
This PR is removing Tailwind (and I agree we shouldn't use it in this example) so we should also change these styles to be pure CSS.
I think these are the only tailwind classes in the example.